suites/encoders-icub.xml

```diff
@@ -4,17 +4,6 @@
     <description> Testing encoders</description>
     <environment>--robotname icub</environment>
 
-    <test type="dll" param="--from optical_encoders_drift_left_arm.ini">  OpticalEncodersDrift </test>
-    <test type="dll" param="--from optical_encoders_drift_right_arm.ini"> OpticalEncodersDrift </test>
-    <test type="dll" param="--from optical_encoders_drift_left_leg.ini">  OpticalEncodersDrift </test>
-    <test type="dll" param="--from optical_encoders_drift_right_leg.ini"> OpticalEncodersDrift </test>
-    <test type="dll" param="--from optical_encoders_drift_torso.ini">     OpticalEncodersDrift </test>
```

Comment:
Also the `optical_encoders` ini files?

Reply:
they were removed for debug only on a part

Reply:
Didn't get this point. Could you elaborate more on that?